### ISSUE TITLE
Missing current temperature reading

### ISSUE THREAD

**Issue by ursusHU** [codyc1515/homebridge-panasonic-air-conditioner#43]
Hi,
The latest version has a small problem. The currently measured temperature isn’t visible.
![6DADF376-6A85-4A03-B697-46A2E23DD64B](https://user-images.githubusercontent.com/61931127/100550160-4b8e1880-3278-11eb-842a-7846e8428a04.png)

**Comment by codyc1515** [comment]
Can you please send a comparison pic from the Comfort Cloud app?

In the interests of reliability the Homebridge plugin now doesn't show the temperature immediately on load. I hope to fix this shortly.

**Comment by HomekitDave** [comment]
Hi @codyc1515 

On this topic; Concealed Panasonic units don't have an internal temperature read-out within the Comfort Cloud app. As a result, my HomeKit displays the temperature from the outdoor unit, which can be confusing, is it possible to simply display the temperature the unit is set to if the unit doesn't have an indoor temperature read-out?

**Comment by ursusHU** [comment]
Hi,

As you can see the internal temperature is present in the Panasonic app, the external is only available when the unit is on. I can test it tomorrow, maybe the 2 values are exchanged, and that’s why I get 0...

![ADDA7C24-EF40-4564-93CB-DECCD602BECD](https://user-images.githubusercontent.com/61931127/100554080-b51b2080-3292-11eb-84e6-b217e08f9108.png)

**Comment by codyc1515** [comment]
Thanks for the feedback. I'll look into this shortly. For reference - my heat pump only provides the outdoors temperature for some reason. I can't see the indoors temperature annoyingly.

**Comment by ursusHU** [comment]
I have checked the valu is not representing the external temperature. Igen still 0, even if in the Panasonic app the value appears.
![158D0232-7F4A-4168-B95A-BE60E376D542](https://user-images.githubusercontent.com/61931127/100624934-438bb280-3324-11eb-9a7f-2a02f8ec6738.png)

**Comment by ursusHU** [comment]
Hi, I just reinstalled my whole Homebridge configuration. And tried to install previous version of your plugin, and suprisingly the login to Comfort Cloud works (with 1.7). I wanted to mention, that this version is showing the internal temperature correctly.

**Comment by arkadicolson** [comment]
I have the same issue. No current temp reading:

[12/28/2020, 6:52:56 PM] [Keuken] {
  deviceGuid: 'CS-TZ50WKEW+4758703378',
  timestamp: 1609177139000,
  summerHouse: 0,
  iAutoX: false,
  nanoe: false,
  autoMode: true,
  heatMode: true,
  fanMode: false,
  dryMode: true,
  coolMode: true,
  ecoNavi: false,
  powerfulMode: true,
  permission: 2,
  quietMode: true,
  airSwingLR: true,
  ecoFunction: 0,
  temperatureUnit: 0,
  modeAvlList: { autoMode: 1, fanMode: 1 },
  parameters: {
    devGuid: 'CS-TZ50WKEW+4758703378',
    eventTime: null,
    online: true,
    operate: 0,
    operationMode: 3,
    temperatureSet: 25,
    fanSpeed: 1,
    fanAutoMode: 3,
    airSwingLR: 2,
    airSwingUD: 0,
    airDirection: 1,
    ecoMode: 0,
    ecoNavi: 0,
    nanoe: 0,
    iAuto: 0,
    defrosting: -255,
    errorCode: null,
    errorCodeStr: null,
    insideTemperature: 21,
    outTemperature: 4,
    devRacCommunicateStatus: -255,
    errorStatus: -255,
    errorStatusFlg: false,
    httpErrorCode: -255,
    actualNanoe: 0,
    airQuality: 0,
    ecoFunctionData: 0,
    updateTime: 1609177139000
  },
  autoTempMax: -1,
  dryTempMax: -1,
  heatTempMax: -1,
  coolTempMax: -1,
  autoTempMin: -1,
  dryTempMin: -1,
  heatTempMin: -1,
  coolTempMin: -1,
  fanSpeedMode: -1,
  fanDirectionMode: -1
}

**Comment by miguelsasso** [comment]
I have a similar issue. Two air con units. Only one showing temperature. Other showing 0C. 

![Uploading 0BD9C8C6-EFCC-4AE1-ABC2-3805F6D0DE5B.jpeg…]()

**Comment by codyc1515** [comment]
Okay so. I must have some kind of basic-tier heat pump because mine does not report the inside temperature (what!?) and it doesn't report the outside temperature when its off. So I added a check recently if the power is off to not change the temperature, otherwise it will go down to zero. I've changed this now to validate the on/off & temperature separately. This should resolve the issue when I publish the fix soon. Sorry about the confusion here.
